src/main/java/graphql/util/Anonymizer.java

```diff
@@ -285,7 +274,8 @@ public TraversalControl visitGraphQLAppliedDirective(GraphQLAppliedDirective gra
                 if (Directives.DEPRECATED_DIRECTIVE_DEFINITION.getName().equals(graphQLDirective.getName())) {
```

Comment:
As I mentioned above, the censoring of `@deprecated` reasons should actually only be here, in the `GraphQLAppliedDirective` visitor method only